spec/lib/salus/scanners/yarn_audit_spec.rb

```diff
@@ -71,7 +71,7 @@
 
       expect(scanner.report.to_h.fetch(:passed)).to eq(false)
       vulns = JSON.parse(scanner.report.to_h[:info][:stdout]).sort { |a, b| a["ID"] <=> b["ID"] }
-      expect(vulns.size).to eq(17)
+      expect(vulns.size).to eq(18)
```

Comment:
there is a new vuln now?  was that added to the vuln DB recently? Just wanted to make sure the changes didn't break anything.

Reply:
I am probably guessing its a new vuln but will check it